db/schema.rb

```diff
@@ -61,7 +61,7 @@
 
   add_index "default_addresses", ["user_id"], name: "index_default_addresses_on_user_id", using: :btree
 
-  create_table "goods", force: true do |t|
+  create_table "goods", force: :cascade do |t|
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.